tests/functional/s/superfluous_parens.py

```diff
@@ -72,6 +65,19 @@ def __iter__(self):
 if (A == 2) is not (B == 2):
     pass
 
+K = ("Test " + "String")  # [superfluous-parens]
 M = A is not (A <= H)
 M = True is not (M == K)
 M = True is not (True is not False)  # pylint: disable=comparison-of-constants
+
+Z = "TestString"
+X = ("Test " + "String")  # [superfluous-parens]
+Y = ("Test " + "String") in Z  # [superfluous-parens]
+assert ("Test " + "String") in "hello"  # [superfluous-parens]
+assert ("Version " + "String") in ("Version " + "String")  # [superfluous-parens]
+
+hi = ("CONST")  # [superfluous-parens]
+hi = ("CONST",)
+
+#TODO: maybe get this line to report [superfluous-parens] without causing other false positives.
+assert "" + ("Version " + "String") in Z
```

Comment:
alright so this is a legitimate case for raising superlous-parens and was brought up in the issue, but I cannot fix it without causing other bigger issues. I think it's probably ok to leave this as a false negative?